src/main/java/net/mcreator/element/BaseType.java

```diff
@@ -38,7 +38,8 @@
 	POTION,
 	POTIONEFFECT,
 	FEATURE,
-	TAB;
+	TAB,
+	VILLAGERTRADE;
 

```

Comment:
Klemen removed some BaseTypes a few times ago, so maybe you should move this one to OTHER or rename it for VILLAGER, so this way we won't have two BaseTypes (when custom villagers will be added) and each is used by one ME only.

Reply:
Ok it's done. I modified the name to `VILLAGER` and both Villager (trade & creator) ME will use this BaseType